### ISSUE TITLE
taup_setsac can not accept a directory name

### ISSUE THREAD

**Issue by seisman** [crotwell/TauP#13]
The TauP documentation says:

> If the given filename is a directory, TauP SetSac will recursively look for files within that directory to process. Thus, a large directory structure of Sac files can be processed easily.

but it doesn't work as expected:

```
taup_setsac -mod iasp91 -ph p-0,s-1 -evdpkm ./SACDIR/
./SACDIR/ is not a file.
Warning: I don't understand the following arguments, continuing:
./SACDIR/
```

**Comment by crotwell** [comment]
Fixed in 17ccee4, and will be in next version, thanks for reporting.

**Comment by crotwell** [comment]
prerelease with this fix in is here:
http://www.seis.sc.edu/downloads/TauP/prerelease/TauP-2.5.0-SNAPSHOT.tgz
